redisvl/utils/utils.py

```diff
@@ -191,3 +191,10 @@ def wrapper():
             return
 
     return wrapper
+
+
+def norm_cosine_distance(value: float) -> float:
```

Comment:
Could add additional check logic to this function kept it simple stupid to start